packages/client/hmi-client/src/components/workflow/tera-calibration.vue

```diff
@@ -39,48 +124,85 @@
 </template>
 
 <script setup lang="ts">
-import { computed, ref, shallowRef, watch } from 'vue';
+import { computed, ref, shallowRef, watch, onMounted } from 'vue';
+import { isEmpty } from 'lodash';
 import Button from 'primevue/button';
 import { makeCalibrateJob, getRunStatus, getRunResult } from '@/services/models/simulation-service';
 import { CalibrationParams, CsvAsset } from '@/types/Types';
 import { ModelConfig } from '@/types/ModelConfig';
 import Dropdown from 'primevue/dropdown';
+import DataTable from 'primevue/datatable';
+import Column from 'primevue/column';
+import Accordion from 'primevue/accordion';
+import AccordionTab from 'primevue/accordiontab';
 import { downloadRawFile } from '@/services/dataset';
 import { PetriNet } from '@/petrinet/petrinet-service';
 import { WorkflowNode } from '@/types/workflow';
 import TeraAsset from '@/components/asset/tera-asset.vue';
+import TeraAssetNav from '@/components/asset/tera-asset-nav.vue';
+import TeraModelDiagram from '@/components/models/tera-model-diagram.vue';
+import TeraModelConfiguration from '@/components/models/tera-model-configuration.vue';
+import TeraDatasetDatatable from '@/components/dataset/tera-dataset-datatable.vue';
+
+enum CalibrationView {
+	INPUT = 'input',
+	OUTPUT = 'output'
+}
 
 const props = defineProps<{
 	node: WorkflowNode;
 }>();
-const modelConfig = computed(() => props.node.inputs[0].value as ModelConfig | undefined);
-const datasetId = computed(() => props.node.inputs[1].value as number | undefined);
 
-const runId = ref(props.node.outputs[0].value as number | undefined);
+const modelConfigurationRef = ref();
```

Comment:
What are the typing on these?

Reply:
This could be done with an HTMLElement type or something similar but they aren't always compatible depending on what you're doing. It's one of those typing hell situations so I am not worrying about it for now.